src/backend/tasks_io/models/notification_requests/fcm_request.py

```diff
@@ -44,12 +52,12 @@ def __init__(self, app, notification, tokens=None, legacy_data_format=False):
         self.tokens = tokens
         self.legacy_data_format = legacy_data_format
 
-    def __str__(self):
+    def __str__(self) -> str:
         return "FCMRequest(tokens={}, notification={})".format(
             self.tokens, str(self.notification)
         )
 
-    def send(self):
+    def send(self) -> messaging.BatchResponse:
```

Comment:
- [ ] The return type here between `FCMRequest` and `WebhookRequest` is different. I'm surprised our linter doesn't catch this. Think about how to standardize this.